evil-collection-git-timemachine.el

```diff
@@ -0,0 +1,22 @@
+;;; evil-collection-git-timemachine.el --- Bindings for `git-timemachine' -*- lexical-binding: t -*-
+;; Author: William Carroll <wpcarro@gmail.com>
+
+;;; Commentary:
+;; Evil keybindings for `git-timemachine' that conform to the principles outlines in evil-collection
+
+;;; Code:
+(require 'evil)
+(require 'git-timemachine)
+
+(defun ecgt--setup-bindings ()
```

Comment:
Here `ecgt` stands for `evil-collection-git-timemachine`. I was unsure of the idioms that `evil-collection` preferred for namespacing, but I did want to ensure that the `--` was present to encode the intention of privacy.

Reply:
You can use the full namespace instead which becomes

evil-collection-git-timemachine--setup-bindings but as mentioned in another comment, you can just remove this function and set up the bindings in the original setup function.